<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Processar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/gear.png</normaloff>images/gear.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/disk.png</normaloff>images/disk.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/door-open-out.png</normaloff>images/door-open-out.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
